articles/tokens/access-token.md

```diff
@@ -9,34 +9,34 @@ toc: true
 
 The Access Token should be used as a **Bearer** credential and transmitted in an HTTP **Authorization** header to the API. 
 
-## Access Token format
+## Access Token Format
 
-Auth0 currently generates Access Tokens in two formats: as opaque strings, or as [JSON Web Tokens (JWTs)](/jwt).
+Auth0 currently generates Access Tokens in two formats: as opaque strings or as [JSON Web Tokens (JWTs)](/jwt).
 
-This depends on the value that the **audience** parameter has in in the [authorization request](/api/authentication#authorize-client).
+This depends on the value that the **audience** parameter has in the [authorization request](/api/authentication#authorize-client).
 
-::: panel What is the audience?
-The **audience** is a parameter set during [authorization](/api/authentication#authorize-client), and it contains the unique identifier of the target API. This is how you tell Auth0 for which API to issue an Access Token, which is the intended *audience* of this token. If you do not want to access a custom API, then by setting the audience to `${account.namespace}/userinfo`, you can use the opaque Access Token to [retrieve the user's profile](/api/authentication#get-user-info).
+::: panel What is the Audience?
+The **audience** is a parameter set during [authorization](/api/authentication#authorize-client), and it contains the unique identifier of the target API. This is how you tell Auth0 for which API to issue an Access Token--which is the intended *audience* of this token. If you do not want to access a custom API, then by setting the audience to `${account.namespace}/userinfo`, you can use the opaque Access Token to [retrieve the user's profile](/api/authentication#get-user-info).
 :::
 
 * If the **audience** is set to `${account.namespace}/userinfo`, then the Access Token will be an opaque string.
 * If the **audience** is set to the unique identifier of a custom API, then the Access Token will be a [JSON Web Token (JWT)](/jwt).
 
 When the **audience** is set to a custom API and the **scope** parameter includes the `openid` value, then the generated Access Token will be a JWT valid for both [retrieving the user's profile](/api/authentication#get-user-info) and for accessing the custom API. The **audience** parameter of this JWT will include two values: `${account.namespace}/userinfo` and your custom API's unique identifier.
 
-:::panel Use RS256 for multiple audiences
-If you set a custom API audience and also use `scope=openid` in your request, then your custom API must use **RS256** (read [how to change an API's settings](/apis#api-settings)). Tokens signed with HS256 can hold only one audience, for security reasons. This applies also if you have set a **Default Audience** at your [API Authorization settings](${manage_url}/#/tenant).
+:::panel Use RS256 for Multiple Audiences
```

Comment:
no reason for capitalization, use sentence case at panels (except for things that are always capitalized, like Lock)